vllm/config/parallel.py

```diff
@@ -294,28 +347,42 @@ def _validate_parallel_config(self) -> Self:
                 "data_parallel_external_lb can only be set when data_parallel_size > 1"
             )
 
+        if not self.enable_expert_parallel:
+            if self.enable_eplb:
+                raise ValueError("enable_expert_parallel must be True to use EPLB.")
+            if self.eplb_config.num_redundant_experts > 0:
+                raise ValueError(
+                    "num_redundant_experts can only be set when "
+                    "enable_expert_parallel is True"
+                )
+            elif self.eplb_config.num_redundant_experts == -1:
+                self.eplb_config.num_redundant_experts = 0
+            return
```

Comment:
```suggestion
            return self
```
I think this is also the cause of the docs failure